tsa-core/src/main/kotlin/org/usvm/machine/interpreter/TvmDictOperationInterpreter.kt

```diff
@@ -1376,6 +1480,35 @@ class TvmDictOperationInterpreter(
         val dictId = DictId(keyLength)
         val resultDict = scope.calcOnState { memory.allocConcrete(TvmDictCellType) }
 
+        val keySort = ctx.mkBvSort(keyLength.toUInt())
+        val allSetEntries =
+            scope.calcOnStateCtx {
+                dictCellRef?.let { memory.setEntries(it, dictId, keySort, DictKeyInfo) }
+            }
+        if (allSetEntries != null && allSetEntries.isInput) {
+            if (getOldValue) {
+                error("unsupported getting old value in input dicts `set` operation")
+            }
+            val dictConcreteRef =
+                dictCellRef as? UConcreteHeapRef
+                    ?: error("unsupported")
```

Comment:
Are you going to support case of ite?

Reply:
I'll postpone the implementation to the other pull request (the corresponding issue is #132)